.gitignore

```diff
@@ -37,3 +37,10 @@
 .env
 /app/assets/builds/*
 !/app/assets/builds/.keep
```

Comment:
I think we can add vendor/assets to git ignore too. Or will this break the application?